### DISCUSSION TITLE
PIMCORE 11 : Object CSV import not available in pimcore 11

### DISCUSSION THREAD

**Discussion by sandyHM** [pimcore/pimcore#17199] [Q&A / Support]
Object CSV import not available in pimcore 11 but it is available in pimcore 6.6 and pimcore 10.  I want to know it is removed or shift somewhere else.
![csv import](https://github.com/pimcore/pimcore/assets/107986459/b15dd58d-e435-4b32-a4e3-9184afbcf269)

**Comment by blankse** [comment]
@sandyHM Pimcore Data Importer https://github.com/pimcore/data-importer is the successor for the CSV Import of Pimcore 6

**Reply by sandyHM** [reply]
Thank you for reply @blankse. we will try.

**Reply by rajeshkumarhm** [reply]
For Pimcore 6 it's available. Missing in Pimcore 11.

**Comment by sandyHM** [comment]
For Pimcore 6 and 10 it's available by default .  why we  install separately in pimcore 11 ?
